presto-hive-hadoop2/bin/run_hive_abfs_tests.sh

```diff
@@ -18,6 +18,34 @@ exec_in_hadoop_master_container sed -i \
     -e "s|%ABFS_ACCOUNT%|${ABFS_ACCOUNT}|g" \
     /etc/hadoop/conf/core-site.xml
 
+# restart hive-server2 to apply S3 changes in core-site.xml
+docker exec "$(hadoop_master_container)" supervisorctl restart hive-server2
+retry check_hadoop
+
+# create test table
+table_path="abfs://${ABFS_CONTAINER}@${ABFS_ACCOUNT}.dfs.core.windows.net/presto_test_external_fs/"
```

Comment:
for s3 tests we leverage auto-delete functionality to provide test isolation
https://github.com/prestosql/presto/blob/cbf0d04b766d7a90c69e8dd7bc18a48779441749/presto-hive-hadoop2/bin/run_hive_s3_tests.sh#L10

Azure has an equivalent: https://stackoverflow.com/a/56906611/65458

@electrum @anusudarsan can we have this enabled?

otherwise I would want this folder to be called `..._2`, to ensure no conflicts in installations running previous versions of these tests, that used different set of test files (one vs four)